res/app/components/stf/common-ui/modals/save-log-modal/save-log-spec.js

```diff
@@ -0,0 +1,11 @@
+describe('SaveLogService', function() {
+
+  beforeEach(angular.mock.module(require('./').name))
+
+  it('should ...', inject(function() {
+
+	//expect(SaveLogService.doSomething()).toEqual('something');
```

Comment:
There should be some tests perhaps

Reply:
> Hi @koral--,
I didn't implemented any test, as I was not able to run jasmine and mocha tests. I will be happy to look at them. Is there any Dockerfile with setup of test environment on which I can start tests?

Reply:
There is nothing more apart from files available in this repo.